src/common/utils/isLastRecord.js

```diff
@@ -0,0 +1,8 @@
+export const isLastRecord = recordRow => {
```

Comment:
why is it moved to common folder? it's used only for marc rows

Reply:
It moved to utils as a common function